Jenkinsfile

```diff
@@ -0,0 +1,92 @@
+
+import groovy.json.JsonSlurper
+// This Jenkinsfile is used by Jenkins to run the BioModels step of Reactome's release.
+// It requires that the OrthoinferenceStableIdentifierHistory step has been run successfully before it can be run.
+def currentRelease
+def previousRelease
+pipeline {
+	agent any
+
+	stages {
+  		// This stage checks that an upstream project, AddLinks-Insertion, was run successfully for its last build.
+		stage('Check OrthoinferenceStableIdentifierHistory build succeeded'){
+			steps{
+				script{
+					currentRelease = (pwd() =~ /Releases\/(\d+)\//)[0][1];
+					previousRelease = (pwd() =~ /Releases\/(\d+)\//)[0][1].toInteger() - 1;
+					// This queries the Jenkins API to confirm that the most recent build of OrthoinferenceStableIdentifierHistory was successful.
+					def OrthoinferenceStableIdentifierHistoryStatusUrl = httpRequest authentication: 'jenkinsKey', validResponseCodes: "${env.VALID_RESPONSE_CODES}", url: "${env.JENKINS_JOB_URL}/job/$currentRelease/job/OrthoinferenceStableIdentifierHistory/lastBuild/api/json"
+					if (OrthoinferenceStableIdentifierHistoryStatusUrl.getStatus() == 404) {
```

Comment:
No need for any change, but just curious if there's a NOT FOUND constant for 404?

Reply:
I'm not sure. This is Groovy, so I didn't look into calling constants or importing libraries that have constants